google-cloud-video-live_stream/samples/get_input.rb

```diff
@@ -0,0 +1,63 @@
+# Copyright 2022 Google, Inc
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+def get_input project_id:, location:, input_id:
+  # [START livestream_get_input]
+  # project_id  = "YOUR-GOOGLE-CLOUD-PROJECT"  # (e.g. "my-project")
+  # location    = "YOUR-INPUT-LOCATION"  # (e.g. "us-central1")
+  # input_id    = "YOUR-INPUT"  # (e.g. "my-input")
+
+  # Require the Live Stream client library.
+  require "google/cloud/video/live_stream"
+
+  # Create a Live Stream client.
+  client = Google::Cloud::Video::LiveStream.livestream_service
+
+  # Build the resource name of the input.
+  name = client.input_path project: project_id, location: location, input: input_id
+
+  # Get the input.
+  input = client.get_input name: name
+
+  # Print the input name.
+  puts "Input: #{input.name}"
+  # [END livestream_get_input]
+
+  input
```

Comment:
we generally do not return values in samples method, showing how to access them should be good

Reply:
I'm confused. The current guidance says to follow the Secret Manager samples (https://screenshot.googleplex.com/BT4Y87v3VRDtV2e). Secret Manager returns values (see https://github.com/googleapis/google-cloud-ruby/blob/main/google-cloud-secret_manager/samples/snippets.rb). Note that I also returned values in the Transcoder samples (https://github.com/googleapis/google-cloud-ruby/commit/5edc40b36938dc58b9974fe98c6f2f2e0e3b6050). Has the guidance changed?

Reply:
yes the sample guidelines have been updated.. please refer this PR for the structure https://github.com/googleapis/google-cloud-ruby/pull/19090/files#diff-c81b6c61172b0db53f3f62f36421248553ba8112b29e32b1f24813446a817312..

Reply:
Refactored